lightspark-sdk/src/commonTest/kotlin/com/lightspark/sdk/ClientIntegrationTests.kt

```diff
@@ -60,6 +61,13 @@ class ClientIntegrationTests {
         println("L1 fee estimate: $estimates")
     }
 
+    @Test
+    fun `get withdrawal fee estimate`() = runTest {
```

Comment:
Mind also adding a test to LightsparkSyncClientTest? That'll help you see how it's called from Java.